cosmos_rl/policy/model/deepseek_v3/configs/deepseek_config_small.py

```diff
@@ -0,0 +1,142 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Would it be possible to make the config full start from config small and only change the fields that are required to change?

Reply:
I've reworked the configs, I've removed the `configs` folder entirely, and now entirely rely on the model config.